inc/search.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Search related functions.
+ *
+ * @package shiro
+ */
+
+/**
+ * Remove post_type = 'profile' from search results.
+ *
+ * @param object $query Full WP_Query object.
+ *
+ * @return void
+ */
+function wmf_remove_profile_from_search( $query ) {
+	if ( $query->is_search() && $query->is_main_query() && ! isset( $_GET['post_type'] ) ) { //phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		$query->set( 'post_type', [
```

Comment:
This makes sense to do, but I wonder if it'd be simpler to just set `exclude_form_search` in the arguments when registering the profile post type?

https://developer.wordpress.org/reference/functions/register_post_type/#exclude_from_search

Reply:
I found appropriate restricting search to posts and pages as client expect, removing stories, profiles and other CPTs that could be added in the future - to appear in the search it needs to be intentionally added.